codespell_lib/data/dictionary.txt

```diff
@@ -9651,6 +10038,7 @@ credists->credits
 creditted->credited
 creedence->credence
 cresent->crescent
+cresh->creche
```

Comment:
It's actually `crèche` in French and hence in English, at least when non-ASCII characters are OK, which is the case in this dictionary (see other fixes).

Reply:
But `creche` is OK too of course.

Reply:
Right. So I've changed it to `crèche`.